packages/slate/test/interfaces/Path/transform/move_node/ends-before-to-ends-equal.js

```diff
@@ -0,0 +1,15 @@
+import { Path } from 'slate'
+
+const path = [0, 3]
+
+const op = {
+  type: 'move_node',
+  path: [0, 0],
+  newPath: [0, 1],
```

Comment:
This doesn't look like "ends equal"? Is the test name wrong?

Reply:
Sorry, I messed up porting code over to this new fork. I just fixed the test. Thanks for pointing that out as this is the most important test.